scanner/package.go

```diff
@@ -302,6 +302,7 @@ type Field struct {
 	Docs
 	Name string
 	Type Type
+	Pos  int
```

Comment:
Maybe `Number` or `Code` will be better here

Reply:
I don't think it makes sense because Pos is already used in protobuf.transformField().